docker-bake.hcl

```diff
@@ -0,0 +1,39 @@
+variable "GO_VERSION" {
+  default = "1.18"
+}
+
+target "_common" {
+  args = {
+    GO_VERSION = GO_VERSION
+    BUILDKIT_CONTEXT_KEEP_GIT_DIR = 1
```

Comment:
Needs explanation why we need this.

Reply:
This is one of the builtin build-args provided by the BuildKit itself. 

* BUILDKIT_CONTEXT_KEEP_GIT_DIR=<bool> trigger git context to keep the .git directory

You can reach out to the official documentation to get more detail about them.

> https://docs.docker.com/engine/reference/commandline/buildx_build/#build-arg

Reply:
Thank you for the explanation. Yes, I know what is this variable. Could you add the comment to docker-bake.hcl about the reason "why" this variable being needed?

Reply:
I'm not sure about that it is necessary thing, so, I'm removing.